### DISCUSSION TITLE
Resource Website Link!

### DISCUSSION THREAD

**Discussion by LucyWhitmore** [now-i-know-my-abcd/docs#75] [Announcements]
[Resource Website Link](https://now-i-know-my-abcd.github.io/docs/intro.html)

**Comment by ebrahimebrahim** [comment]
(The link is broken and perhaps meant to be https://now-i-know-my-abcd.github.io/docs/intro.html)
((And thanks for the resource!))

**Reply by LucyWhitmore** [reply]
Thanks so much for catching that! It looks like the edits are saving a little funky, please let us know if you run into an issue later on, we really appreciate the community involvement :)
